core/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -134,19 +134,16 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                     "/habit/statistic/todayStatisticsForAllHabitItems",
                     "/place/about/{id}",
                     "/specification",
-                    "/econews",
-                    "/econews/newest",
-                    "/econews/tags",
-                    "/econews/tags/all",
-                    "/econews/recommended",
-                    "/econews/{id}",
-                    "/econews/countLikes/{econewsId}",
-                    "/econews/comments/replies/{parentCommentId}",
-                    "/econews/comments/count/comments/{ecoNewsId}",
-                    "/econews/comments/count/replies/{parentCommentId}",
-                    "/econews/comments/count/likes",
-                    "/econews/comments/replies/active/{parentCommentId}",
-                    "/econews/comments/active",
+                    ECO_NEWS,
+                    ECO_NEWS + "/tags",
+                    ECO_NEWS + ECO_NEWS_ID + "/recommended",
+                    ECO_NEWS + ECO_NEWS_ID,
+                    ECO_NEWS + ECO_NEWS_ID + "/likes/count",
+                    ECO_NEWS + ECO_NEWS_ID + "/dislikes/count",
+                    ECO_NEWS_COMMENTS,
+                    ECO_NEWS_COMMENTS + PARENT_COMMENT_ID + "/replies",
+                    ECO_NEWS_COMMENTS + PARENT_COMMENT_ID + "/replies/count",
+                    ECO_NEWS_COMMENTS + "/count",
```

Comment:
any reason you moved only `eco news` endpoints to the constant?
Should we move other endpoints to the constants too?